Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -562,20 +562,20 @@ public class PBXProjGenerator {
     }
     
     func getSources(sourceMetadata source: Source, path: Path, depth: Int = 0) throws -> (sourceFiles: [SourceFile], groups: [PBXGroup]) {
-        // if we have a file, move it to children and use the parent as the path
         let (children, path) = path.isFile ?
             ([path], path.parent()) :
```

Comment:
I guess no-one would specify a file path and then exclude it. So not taking care of the file case is fine here 👍